toolchain/gnu/Makefile

```diff
@@ -0,0 +1,232 @@
+TOPDIR = $(CURDIR)
+
+include $(TOPDIR)/common.mk
+
+all: package
```

Comment:
`all: ...` should appear before any Makefile includes. Please note that the default target is the first target which make encounters while scanning the file.